JitsiConference.js

```diff
@@ -730,6 +730,31 @@ JitsiConference.prototype.leave = async function() {
     }
 };
 
+/**
+ * Returns <tt>true</tt> if end conference support is enabled in the backend.
+ *
+ * @returns {boolean} whether end conference is supported in the backend.
+ */
+JitsiConference.prototype.isEndConferenceSupported = function() {
+    return Boolean(this.room && this.room.xmpp.endConferenceComponentAddress);
+};
+
+/**
+ * Ends the conference.
+ */
+JitsiConference.prototype.end = async function() {
```

Comment:
This function is not async, doesn't return a promise or await anything

Reply:
Should we protect ourselves from running this from a breakout room? I'd say we do. Also, what should happen if there are active breakout rooms?

Reply:
> This function is not async, doesn't return a promise or await anything

Good catch. Thanks.

Reply:
> Should we protect ourselves from running this from a breakout room? I'd say we do. Also, what should happen if there are active breakout rooms?

We had the end meeting button hidden on the client while in a breakout room. But I totally agree that we should handle this case here, too. 

The breakout room module is designed to end the breakout rooms, when the main room gets destroyed. So participants in breakout rooms will get the same notification that the conference has ended as those in the main room. Of course the moderator can decide to gracefully close the breakout rooms beforehand, if he would like to announce to everyone that the conference will be ended. Does this make sense?

Reply:
I think it does, yeah!